internal/suites/example/compose/nginx/portal/nginx.conf

```diff
@@ -25,7 +25,7 @@ http {
         error_page 497 301 =307 https://$host:$server_port$request_uri;
 
         # Serve the backend API for the portal.
-        location /api {
+        location ~ ^/(api|oidc|) {
```

Comment:
I thought there would be /api/oidc/..., I did not think it would be at the root of the API. I feel like it makes the config more complex for no gain from Authelia's perspective, so I'd rather keep it under /api/oidc. Sorry if I misunderstood from the beginning.